engine/app/components/citizens_advice_components/input.rb

```diff
@@ -83,7 +84,7 @@ def base_input_attributes
         type: (@type.to_s.dasherize if @type.present?),
         id: input_id,
         name: name,
-        value: value,
+        value: @value,
```

Comment:
This already seemed to come from the above (Line27 then Line5). This also seems to be getting overwritten potentially.

Reply:
Oh yeah! Wait were we already supporting the `value` through `options`?

Reply:
😬  Let me check

Reply:
Well yeah, I've lost the plot 😵  I didn't believe in past-David.
The unit tests still stand so I'll just update this PR to add those.

Reply:
Great spot - thanks @luke-hill  😄